compiler/pipes/sort-and-inherit-classes.cpp

```diff
@@ -384,7 +384,7 @@ void SortAndInheritClassesF::check_on_finish(DataStream<FunctionPtr> &os) {
       );
     }
 
-    if (!c->is_builtin() && c->is_polymorphic_class()) {
+    if ((!c->is_builtin() || c->need_generated_stub) && c->is_polymorphic_class()) {
```

Comment:
Same question about do we need additional checks in such cases

Reply:
Same as above